uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar.component.js

```diff
@@ -267,7 +267,8 @@ define([
       return {
         index: dateMoment.month(),
         year: dateMoment.year(),
-        name: dateMoment.format('MMM')
+        name: dateMoment.format('MMM'),
+        nameFull: dateMoment.format('MMMM')
```

Comment:
* the `name` property should be `MMMM` (full name).
* remove `nameFull`.
* add `shortName` with format `MMM`.

This will require changes on the view.

Reason: a month name is `August`, not `Aug`. `Aug` is the short name of the month. This will make property names less confusing.